tests/bearlib/abstractions/LinterTest.py

```diff
@@ -177,24 +184,24 @@ def test_decorator_invalid_parameter_types(self):
             linter('some-executable',
                    output_format='regex',
                    output_regex='(?P<message>)',
-                   result_message=None)
+                   result_message=None)(object)
 
         with self.assertRaises(TypeError):
             linter('some-executable',
                    output_format='corrected',
-                   result_message=list())
+                   result_message=list())(object)
 
         with self.assertRaises(TypeError) as cm:
             linter('some-executable',
                    output_format='corrected',
-                   diff_severity=999888777)
+                   diff_severity=999888777)(object)
```

Comment:
ah but you should maybe use the test base classes defined in the test-suite, `object` has problems. If you need an empty class, you can use `EmptyTestLinter`. There are a few others around if needed.